templates/inspec/plural_resource.erb

```diff
@@ -46,7 +46,7 @@ link_query_items = object&.nested_query&.keys&.first || object.collection_url_ke
 
   def fetch_wrapped_resource(wrap_path)
     # fetch_resource returns an array of responses (to handle pagination)
-    result = @connection.fetch_all(product_url, resource_base_url, @params)
+    result = @connection.fetch_all(product_url, resource_base_url, @params, 'Get')
```

Comment:
Should this be titlecase or all caps?

Reply:
Titlecase as ruby's net library seems to use titlecase for the constants. 

Closing this in favor of https://github.com/GoogleCloudPlatform/magic-modules/pull/2292 which includes this

Reply:
thanks, i hate it